### DISCUSSION TITLE
Can you add extensions using LTBEEF?

### DISCUSSION THREAD

**Discussion by mattmanz1** [3kh0/ext-remover#40] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is possible to add extensions? if so how? this is cool and all but i want something like a proxy or games.

**Comment by Aspaceman527** [comment, marked as answer]
no, it is not

**Comment by ewrgwerjgwe** [comment]
you can't

**Comment by amelia-bartlett** [comment]
not by default

you might be able to patch it, it depends on what APIs are exposed to the chrome webstore, and if they respect chrome policies for extension install

**Reply by Aka-but-nice** [reply]
Yes, chrome.webstorePrivate.beginPrivateInstallWithManifest3()

**Reply by BroJac5246** [reply]
> Yes, chrome.webstorePrivate.beginPrivateInstallWithManifest3()

Does that follow the extension allowList?
